Sources/KeyboardShortcuts/Shortcut.swift

```diff
@@ -347,25 +719,34 @@ extension KeyboardShortcuts.Shortcut: CustomStringConvertible {
 	//=> "⌘A"
 	```
 	*/
+
+	@MainActor
+	var presentableDescription: String {
+		if let key, let specialKey = keyToSpecialKeyMapping[key] {
+			return modifiers.presentableDescription + specialKey.presentableDescription
+		}
+		return modifiers.presentableDescription + String(keyToCharacter() ?? "�").capitalized
+	}
+
 	@MainActor
 	public var description: String {
-		// We use `.capitalized` so it correctly handles “⌘Space”.
-		modifiers.presentableDescription + (keyToCharacter()?.capitalized ?? "�")
+		// TODO: `description` needs to be `nonisolated`
+		presentableDescription
```

Comment:
Anticipating `SWIFT_STRICT_CONCURRENCY` here.

Solution would seem to be switching the UI to use `presentableDescription` while also changing to `CustomDebugStringConvertible` and having `debugDescription` avoid `keyToCharacter()` by handling `kVK_ANSI_*` directly? It doesn’t look like there’s any way around compromising the API.

Reply:
Maybe open an issue about this? I don't intend to enable strict concurrency anytime soon.